spec/request/disability_claims_spec.rb

```diff
@@ -37,4 +37,14 @@
       expect(response).to have_http_status(:internal_server_error)
     end
   end
+
+  context 'with an outage' do
+    before do
+      EVSS::ClaimsService.breakers_service.begin_forced_outage!
```

Comment:
@austin-martinez Ok I made the change you requested. As you'll see, it's a little janky because:

* I still have to create a ClaimsService, since BaseService is abstract, and
* Since get and post are protected I have to use send to call them on the service.

I'm fine with it this way, though, if you are.